front/src/common/BootstrapSNCF/MultiSelectSNCF.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useMemo, useState } from 'react';
+
+type MultiSelectSNCFProps = {
+  multiSelectTitle: string;
+  multiSelectSubTitle?: string;
+  selectOptions: { label: string }[];
+  onChange: React.Dispatch<React.SetStateAction<string[]>>;
+  selectedValues: string[];
+};
+
+const MultiSelectSNCF = ({
```

Comment:
Why not simply MultiSelect ? 
```suggestion
const MultiSelect = ({
```

Reply:
The component comes from bootstrap SNCF